homeassistant/components/velbus/quality_scale.yaml

```diff
@@ -0,0 +1,70 @@
+rules:
+  # Bronze
+  action-setup: todo
+  appropriate-polling:
+    status: exempt
+    comment: |
+      This integration does not do polling.
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
```

Comment:
how is config-flow todo, while test-coverage is done?

Reply:
the config flow is only missing data_descriptions